convert-junit4-to-junit5/pom.xml

```diff
@@ -127,10 +129,15 @@
 			<version>2.5</version>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>junit</groupId>
+			<artifactId>junit</artifactId>
+			<version>4.12</version>
+		</dependency>
 		<dependency>
 			<groupId>com.github.javaparser</groupId>
 			<artifactId>javaparser-core</artifactId>
-			<version>3.9.0</version>
+			<version>3.8.3</version>
```

Comment:
Downgraded to 3.8.3 because Indentation in later versions [is broken](https://github.com/javaparser/javaparser/pull/2027) if you try to preserve the formatting.

Reply:
I suggest to add this in a comment in `pom.xml`

Reply:
Added a comment. There are also tests that will fail unless the formatting bug is fixed.